addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4222,6 +4222,10 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
             } else {
                 shapeToSelect = possibleShapes[1];
             }
+            this.trigger_up('snippet_edition_request', {exec: () => {
+                // options for shape will only be available after _toggleShape() returned
+                this._requestUserValueWidgets('bg_shape_opt')[0].$el.find('we-toggler')[0].click();
```

Comment:
Can't you use the `enable` widget method instead?

Reply:
Added `enable` on `SelectPagerUserValueWidget` so that it triggers the toggler.